uber/site_sections/departments.py

```diff
@@ -192,6 +193,38 @@ def dept_requests_export(self, out, session, department_id, requested_any=False,
 
             out.writerow(row)
 
+    @csv_file
+    def overworked_attendees(self, out, session):
+        def single_sequence(attendee, start_hour, hour_map):
+            hour_limit = 1000
+            consecutive_hours = 0
+            current_hour = start_hour
+            while current_hour in hour_map:
```

Comment:
This looks really similar to the code you wrote in `working_limit_ok` on the Attendee class -- was there a particular reason to repeat the same code? Just to save time?

Reply:
For those following via GitHub, there's a bunch of minor differences between these functions that add up.